### ISSUE TITLE
web instance stacking

### ISSUE THREAD

**Issue by kxiansen** [openark/orchestrator#1365]
![image](https://user-images.githubusercontent.com/43315194/121138983-d7f24700-c86a-11eb-9584-766d968209d8.png)
Look, it's piled together

**Comment by shlomi-noach** [comment]
what does `orchestrator-client -c topology ...` show?

**Comment by freiman** [comment]
Hi, I have the same problem. There is anonymized command output you want, but for my instance:
```
-> orchestrator-client -c topology -i master:3306
- master:3306    [null,nonreplicating,8.0.20-11,rw,ROW,>>,GTID]
  + slave1:3306  [0s,ok,8.0.20-11,ro,nobinlog,GTID]
  + slave2:3306  [0s,ok,8.0.20-11,ro,nobinlog,GTID]
  + slaveb1:3306 [1897s,lag,8.0.20-11,ro,nobinlog,GTID]
```
Thanks for help

**Comment by shlomi-noach** [comment]
@freiman I see your current primary (`master:3306`) is still configured as replica.
when you do `show slave status\G` on your current primary, what's the `Master_host`? And is there a further connection between that host and the topology, anything circular?

**Comment by freiman** [comment]
yes it is configured as replica and replication is stopped. it is state from past where we manually switching master because some maintanence work.

now we newly added this replication to orchestrator because we want to improve HA for our masters. so in near future wa want to add second master and complete "co master" topology.

so it is in the process.

the strange thing is that this replication is second we added in orchestrator and we had no this problem with the first one where there was also master with non working replication. orchestrator drew replication topology fine.

**Comment by shlomi-noach** [comment]
Thank you for explaining. I'm not sure the specific question was addressed: 

> when you do show slave status\G on your current primary, what's the Master_host? And is there a further connection between that host and the topology, anything circular?

**Comment by freiman** [comment]
yes! problem is solved. it was because master_host had VIP address which is actually set on the master server. so replication on master was configured to replicating from itself. I changed to first slave and diagram is fine.
thank you very much for your help!

**Comment by shlomi-noach** [comment]
Cheers. I'm quite certain the original author of the issue has a similar circular issue.
